web/src/pages/Route/constants.ts

```diff
@@ -165,14 +165,14 @@ export const DEBUG_BODY_TYPE_SUPPORTED: RouteModule.DebugBodyType[] = [
 ];
 
 // Note: codemirror mode: apl for text; javascript for json(need to format); xml for xml;
-export const DEBUG_BODY_CODEMIRROR_MODE_SUPPORTED = [
-  { name: 'JSON', mode: 'javascript' },
-  { name: 'TEXT', mode: 'apl' },
+export const DEBUG_BODY_MODE_SUPPORTED = [
+  { name: 'JSON', mode: 'json' },
+  { name: 'TEXT', mode: 'text' },
```

Comment:
LINE# 167 This comment may need to change or remove.

Reply:
updated.